bench/wtperf/wtperf.c

```diff
@@ -313,6 +315,9 @@ worker_async(void *arg)
 			if (wtperf_value_range(cfg) < next_val)
 				continue;
 			break;
+		case WORKER_TRUNCATE:
```

Comment:
It would be better to catch an attempt to setup truncate and async ops in the configuration parsing, and return an error, and not have any truncate code in this async loop. Rather than having a partial no-op implementation here.

i.e: If I configure async and truncate, I want to get an error on starting wtperf, rather than it turning the configuration into a no-op.